modules/data-entry/src/main/frontend/src/vocabQuery/BrowseVocabularyBranch.jsx

```diff
@@ -46,8 +46,8 @@ import { REST_URL, MakeRequest } from "./util.jsx";
 //
 // Optional arguments:
 //  fullscreen: whether or not the dialog is fullscreen (default: false)
-function ListChild(props) {
-  const { classes, defaultOpen, id, path, name, changeTerm, registerInfo, getInfo, expands, headNode, bolded, onError, knownHasChildren } = props;
+function VocabularyBranch(props) {
+  const { classes, defaultOpen, id, path, name, onTermFocus, registerInfo, getInfo, expands, headNode, bolded, onError, knownHasChildren } = props;
```

Comment:
Rename `bolded` to `focused`

```suggestion
  const { classes, defaultOpen, id, path, name, onTermFocus, registerInfo, getInfo, expands, headNode, focused, onError, knownHasChildren } = props;
```